### PR TITLE
Prepare v0.27.0 release

### PR DESCRIPTION
**Pull request by humivo**
**Description:** bump version and create release notes.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/releases/v0.27.0.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- Add dependabot PR merger script tool [\#1872](https://github.com/aws-observability/aws-otel-collector/pull/1872) ([rapphil](https://github.com/rapphil))
+- Merging release/v0.26.x to main [\#1870](https://github.com/aws-observability/aws-otel-collector/pull/1870) ([PaurushGarg](https://github.com/PaurushGarg))
+- \[Chore\] - Merge release v0.25.x to main [\#1851](https://github.com/aws-observability/aws-otel-collector/pull/1851) ([vasireddy99](https://github.com/vasireddy99))
+- \[Chore\] - Merge release v0.26.x to main [\#1850](https://github.com/aws-observability/aws-otel-collector/pull/1850) ([vasireddy99](https://github.com/vasireddy99))
+- Fix sed command in bump version script [\#1827](https://github.com/aws-observability/aws-otel-collector/pull/1827) ([bryan-aguilar](https://github.com/bryan-aguilar))
+- \[chore\] merge relase/v0.26.x onto main [\#1815](https://github.com/aws-observability/aws-otel-collector/pull/1815) ([bryan-aguilar](https://github.com/bryan-aguilar))
+- Update Readme links for docs [\#1812](https://github.com/aws-observability/aws-otel-collector/pull/1812) ([vasireddy99](https://github.com/vasireddy99))
```
Comment (bryan-aguilar) [inline docs/releases/v0.27.0.md:22]: These can be removed. They are not necessary for the release notes.

File: docs/releases/v0.27.0.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Changelog
+
+## [v0.27.0](https://github.com/aws-observability/aws-otel-collector/tree/v0.27.0) (2023-03-02)
+
+[Full Changelog](https://github.com/aws-observability/aws-otel-collector/compare/v0.25.1...v0.27.0)
+
+**Closed issues:**
+
+- Support for kubernetes 1.24 and dockershim deprecation [\#1769](https://github.com/aws-observability/aws-otel-collector/issues/1769)
```
Comment (bryan-aguilar) [inline docs/releases/v0.27.0.md:9]: This can be removed

File: docs/releases/v0.27.0.md (hunk 1 above)
Comment (humivo) [reply docs/releases/v0.27.0.md]: Ok I have fixed it in latest commit

File: docs/releases/v0.27.0.md (hunk 2 above)
Comment (humivo) [reply docs/releases/v0.27.0.md]: Removed in latest commit

File: docs/releases/v0.27.0.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+## [v0.27.0](https://github.com/aws-observability/aws-otel-collector/tree/v0.27.0) (2023-03-02)
+
+[Full Changelog](https://github.com/aws-observability/aws-otel-collector/compare/v0.25.1...v0.27.0)
+
+**Merged pull requests:**
+
+- Update all upstream dependencies to 0.72.0 and breaking changes [\#1889](https://github.com/aws-observability/aws-otel-collector/pull/1889) ([humivo](https://github.com/humivo))
```
Comment (bryan-aguilar) [inline docs/releases/v0.27.0.md:9]:
```suggestion
- Update upstream OpenTelemetry dependencies to 0.72.0 [\#1889](https://github.com/aws-observability/aws-otel-collector/pull/1889) ([humivo](https://github.com/humivo))
```

**Review by bryan-aguilar — APPROVED** [review]

**Pull request merged**